src/components/rating/rating.tsx

```diff
@@ -304,7 +304,7 @@ export class Rating
                       </div>
                     )}
                     <span class="visually-hidden">
-                      {this.messages.stars.replace("${num}", `${value}`)}
+                      {this.messages.stars.replace("{num}", `${value}`)}
```

Comment:
This might break the localization for all the locales ( excluding `en`) if installed into master. Changes should include all the message bundles. Applies to all the components in PR.